### PR TITLE
Feature/editorjs component

### PR DESCRIPTION
**Pull request by fantasyecf**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,9935 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (brucexu-eth) [inline yarn.lock:1]: Please remove yarn.lock

File: app/spaces/[spaceid]/page.tsx [hunk 2]
```diff
@@ -0,0 +1,618 @@
...
+            ></SidebarButton>
+            <SidebarButton
+              sx={{
+                padding: '10px',
+                borderRadius: '10px',
+                backgroundColor: '#ffffff0a',
+                '&:hover': {
+                  backgroundColor: '#ffffff1a'
+                },
+                cursor: 'pointer'
```
Comment (brucexu-eth) [inline app/spaces/[spaceid]/page.tsx:169]: Duplicate code

File: yarn.lock (hunk 1 above)
Comment (fantasyecf) [reply yarn.lock:1]: Will add it in gitignore.

File: app/spaces/[spaceid]/page.tsx (hunk 2 above)
Comment (fantasyecf) [reply app/spaces/[spaceid]/page.tsx:169]: This is because, it is rebased from feature/event-ticket-home-mobile branch.

File: yarn.lock (hunk 1 above)
Comment (fantasyecf) [reply yarn.lock:1]: Yes, I got it.

**Pull request merged**